client/src/app/shared/components/sort-filter-bar/sort-filter-bar.component.ts

```diff
@@ -134,6 +134,12 @@ export class SortFilterBarComponent<V extends BaseViewModel> {
     }
 
     public set sortOption(option: OsSortingOption<V>) {
+        // If the option has a custom sorting function
+        if (option.sortFn) {
```

Comment:
`this.sortService.sortFn = option.sortFn || null;` if you want to make sure, that it is null and not `undefined`, `0`, ...

Reply:
YOu can remove the complete if-clause and replace it with this one line.